### PR TITLE
fix: Across V1 event clients need to use paginated event search query now that DepositRelayed has over 10,000 events

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

Some nodes like Infura cannot return more than 10,000 events, and the default search for these clients is set to look up events for ALL of history

**Summary**

Sets a default page size for event queries. Note that `InsuredBridgePriceFeed.getHistoricalPrice` is not used in `insured-bridge-relayer` and its only used in the validation script `getHistoricalPrice`.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [X]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ export class InsuredBridgeL1Client {
     readonly rateModelStoreAddress: string | null,
     readonly startingBlockNumber = 0,
     readonly endingBlockNumber: number | null = null,
-    readonly blocksPerEventSearch: number | null = null
+    readonly blocksPerEventSearch: number | null = 20000
```
Comment (nicholaspai) [inline packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts:106]: This shouldn't break V1 bots in prod but worth thinking about

File: packages/financial-templates-lib/src/price-feed/InsuredBridgePriceFeed.ts [hunk 2]
```diff
@@ -69,10 +69,13 @@ export class InsuredBridgePriceFeed extends PriceFeedInterface {
       depositHash: string;
     }
     let matchedRelay: MatchedRelay | undefined;
+    const latestL1Block = await this.l1Client.l1Web3.eth.getBlockNumber()
```
Comment (nicholaspai) [inline packages/financial-templates-lib/src/price-feed/InsuredBridgePriceFeed.ts:72]: This pricefeed is only used in `getHistorifcalPrice.js` script I believe but pls double check me

File: packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts (hunk 1 above)
Comment (nicholaspai) [reply packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts]: I ended up just removing and it still worked

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**